pkg/fluentd/fluentd.go

```diff
@@ -155,7 +155,7 @@ func CreateFluentdContainer(bindingName string, componentName string) corev1.Con
 			},
 			{
 				Name:  "ELASTICSEARCH_HOST",
-				Value: fmt.Sprintf("vmi-%s-es-ingest.%s.svc.cluster.local", bindingName, constants.VerrazzanoNamespace),
+				Value: fmt.Sprintf("vmi-%s-es-ingest.%s.svc.cluster.local", util.GetProfileBindingName(bindingName), constants.VerrazzanoNamespace),
```

Comment:
Could the value of util.GetProfileBindingName(bindingName) be passed into the function instead of passing bindingName?

Reply:
do you mean move that change up a level? to have all callers do it instead of just here?

Reply:
Or are you suggesting that we simply have a local set to the function value within the CreateFluentdContainer?

Reply:
I am suggesting to move the change up one level.  We had a brown bag discussion a couple of months about coding and testing that @markxnelson  led.  One of the discussions was that the testing and stability of a function can be improved if parameters can be passed in as values.  Less can change in the function, so the testing can be more predictable.  That is why I was asking if this method could instead receive the value to use instead of computing it.

Reply:
ok, that sounds like it just pushes the same concern up one level. Pinging separately to discuss if maybe the way to go is to either change the bindingName up front or have an additional profileBindingName which the specific callers can use